README.md

```diff
@@ -0,0 +1,87 @@
+# Cloudant Python Client
+
+[![Build Status](https://travis-ci.org/cloudant/python-cloudant.svg?branch=master)](https://travis-ci.org/cloudant/java-cloudant)
+[![Readthedocs](https://readthedocs.org/projects/pip/badge/)](http://python-cloudant.readthedocs.io)
+[![Compatibility](https://img.shields.io/badge/python-2.7%2C%203.5-blue.svg)](http://python-cloudant.readthedocs.io/en/latest/compatibility.html)
+[![pypi](https://img.shields.io/pypi/v/cloudant.svg)](https://pypi.python.org/pypi/cloudant)
+
+This is the official Cloudant library for Python.
+
+* [Installation and Usage](#installation-and-usage)
+* [Getting Started](#getting-started)
+* [API Reference](http://www.javadoc.io/doc/com.cloudant/cloudant-client/)
```

Comment:
Needs to point to the sub-section of readthedocs not javadoc.io

Reply:
Fixed in 59d0747.